setup.py

```diff
@@ -10,7 +10,7 @@
 
 setup(
     name='q2-coordinates',
-    version='2020.8.1.dev',
+    version='2021.2.0.dev',
```

Comment:
If we want q2-coordinates to be up-to-date with the latest Qiime 2 then this version should be set to `2021.4.0` as this is the one that will get released soon.

Reply:
this PR and linked issue #20 is aiming to update the plugin from `2020.11.1` to `2021.2.0`. Issue #19 will update from `2021.2.0` to `2021.4.0`.